### PR TITLE
Headroom added

### PR DESCRIPTION
**Pull request by archit0210**

### CODE REVIEW

File: src/components/header/Header.js [hunk 1]
```diff
@@ -2,12 +2,15 @@ import React from "react";
...
 import {greeting, workExperiences} from "../../portfolio";
+import { headroom } from "headroom";
 
 function Header() {
   const exp = workExperiences.viewExperiences;
   return (
-    <Fade top duration={1000} distance="20px">
-    <div>
+  <Fade top duration={1000} distance="20px">
+    <script src="https://unpkg.com/headroom.js"></script>
```
Comment (naveen521kk) [inline src/components/header/Header.js:11]: You should not use script tag here.

File: src/components/header/Header.js [hunk 2]
```diff
@@ -48,6 +51,9 @@ function Header() {
         </ul>
       </header>
     </div>
+    <script>
```
Comment (naveen521kk) [inline src/components/header/Header.js:54]: Again this also.

File: src/components/header/Header.js (hunk 1 above)
Comment (naveen521kk) [reply src/components/header/Header.js:11]: Instead here
```html
<Headroom>
```
Should be coming.

Comment (archit0210) [reply src/components/header/Header.js:11]: but the import headroom from headroom is not working

Comment (kartik918) [reply src/components/header/Header.js:11]: I tried to guide you in my comment on #174 too, take a look at Headroom's documentation.

Comment (archit0210) [reply src/components/header/Header.js:11]: i added and followed all the steps and this error occurred
Capture

Comment (kartik918) [reply src/components/header/Header.js:11]: Change Headroom.js to `"Headroom"` @archit0210

Comment (archit0210) [reply src/components/header/Header.js:11]: i did, there is no change in error

Comment (archit0210) [reply src/components/header/Header.js:11]: okay, now is this correct?

Comment (archit0210) [reply src/components/header/Header.js:11]: okay, now is this correct?

Comment (kartik918) [reply src/components/header/Header.js:11]: Umm, why are you initializing Headroom, I don't remember exactly but it isn't as complicated as your implementation, you just need to import Headroom and wrap the Header component in it. 
Headroom will make the header sticky itself!

Comment (archit0210) [reply src/components/header/Header.js:11]: oh sorry i forgot, i have wrapped header component...

Comment (archit0210) [reply src/components/header/Header.js:11]: i tried restarting and checked the names of files but couldn't resolve!!

**Comment by kartikchou** [comment]
Hey @archit0210 I see you're struggling with this so I'm gonna take this issue up and open a PR for it, maybe it'll help clear much of your confusion. Is that okay?

File: src/components/header/Header.js (hunk 1 above)
Comment (archit0210) [reply src/components/header/Header.js:11]: No

**Comment by kartikcho** [comment]
Superseded by #209 , @archit0210 you might want to take a look!

**Pull request closed without merging**